cedar-policy-validator/src/schema_file_format.rs

```diff
@@ -474,10 +482,8 @@ pub enum SchemaTypeVariant {
         element: Box<SchemaType>,
     },
     Record {
-        #[serde(with = "serde_with::rust::maps_duplicate_key_is_error")]
         attributes: BTreeMap<SmolStr, TypeOfAttribute>,
         #[serde(rename = "additionalAttributes")]
-        #[serde(default = "additional_attributes_default")]
```

Comment:
why remove this?  and if we're removing this, should we also remove the function it refers to?

Reply:
It's still called by the manual deserialization implementation. It's removed because the annotation is dead code since were not deriving deserialize.